### ISSUE TITLE
Invalid Operation Exception

### ISSUE THREAD

**Issue by garlantm** [nemosminer/NemosMiner-v2.4.1#216]
Hello,

Does This require that i run it on a 64bit windows? 
Is there a workaround that would let me run it on 32bit if this is the case?
Thank you,
Trent

**********************
Windows PowerShell transcript start
Start time: 20180123205849
Username: DESKTOP-RFL22US\miner2560
RunAs User: DESKTOP-RFL22US\miner2560
Configuration Name: 
Machine: DESKTOP-RFL22US (Microsoft Windows NT 10.0.16299.0)
Host Application: powershell -version 5.0 -noexit -executionpolicy bypass -windowstyle maximized -command &.\NemosMiner-v2.3.ps1 -SelGPUDSTM '0,1' -SelGPUCC '0,1' -Currency USD -Passwordcurrency BTC -interval 120 -Username garlantm -Workername Trent1 -Location US -PoolName miningpoolhub -Type nvidia -Algorithm skein,lyra2z,equihash -Donate 0
Process ID: 5500
PSVersion: 5.1.16299.98
PSEdition: Desktop
PSCompatibleVersions: 1.0, 2.0, 3.0, 4.0, 5.0, 5.1.16299.98
BuildVersion: 10.0.16299.98
CLRVersion: 4.0.30319.42000
WSManStackVersion: 3.0
PSRemotingProtocolVersion: 2.3
SerializationVersion: 1.1.0.1
**********************
Transcript started, output file is .\Logs\2018-01-23_20-58-48.txt
Loading BTC rate from 'api.coinbase.com'..
Loading pool stats..
Loading miners..
This command cannot be run due to the error: The specified executable is not a valid application for this OS platform..
    + CategoryInfo          : InvalidOperation: (:) [Start-Process], InvalidOperationException
    + FullyQualifiedErrorId : InvalidOperationException,Microsoft.PowerShell.Commands.StartProcessCommand
    + PSComputerName        : localhost
This command cannot be run due to the error: The specified executable is not a valid application for this OS platform..
    + CategoryInfo          : InvalidOperation: (:) [Start-Process], InvalidOperationException
    + FullyQualifiedErrorId : InvalidOperationException,Microsoft.PowerShell.Commands.StartProcessCommand
    + PSComputerName        : localhost

**Comment by jagauthier** [comment]
Looks like the miners that are being downloaded are x86.  You could download them yourself, and place them in the correct Bin folder.

**Comment by garlantm** [comment]
Hello,

Thank you for your response. I ended up going back to windows 7 so I could install 64 bit windows and it solved the problem (for some stupid reason windows 10 wouldnt let me install 64 bit). 

the interesting part is that it would work as long as I only ran one type of miner (normally skein) it would chug along all day without any problems. But as soon as you added a second miner it was done!

Thank you for the response!
